cmake/modules/SwiftSource.cmake

```diff
@@ -499,15 +508,11 @@ function(_compile_swift_files
     add_custom_command_target(
         module_dependency_target
         COMMAND
-          "${CMAKE_COMMAND}" "-E" "remove" "-f" "${module_file}"
-        COMMAND
-          "${CMAKE_COMMAND}" "-E" "remove" "-f" "${module_doc_file}"
-        COMMAND
-          "${CMAKE_COMMAND}" "-E" "remove" "-f" "${interface_file}"
+          "${CMAKE_COMMAND}" "-E" "remove" "-f" ${module_outputs}
```

Comment:
If this works, that is great.  I recently learnt that the multi-file variant of the commands was added in CMake 3.5 for some of them :/

Reply:
Oh, hm. It works on my machine™ but I'll go check before I merge.

Reply:
Looks like it was [in 3.4](https://cmake.org/cmake/help/v3.4/manual/cmake.1.html#command-line-tool-mode) at least, and our current oldest supported version is 3.4.3.